tests/lib/rules/no-invalid-template-root.js

```diff
@@ -50,16 +50,23 @@ tester.run('no-invalid-template-root', rule, {
     {
       filename: 'test.vue',
       code: `<template>\n    <c1 v-if="1" />\n    <c2 v-else-if="1" />\n    <c3 v-else />\n</template>`
-    }
-  ],
-  invalid: [
+    },
     {
       filename: 'test.vue',
-      code: '<template>\n</template>',
-      errors: ['The template root requires exactly one element.']
+      code: '<template>\n</template>'
```

Comment:
@HerringtonDarkholme Could you know where the document about `<template src="foo.html">` is? [User guide](https://vuejs.org/v2/guide/single-file-components.html#What-About-Separation-of-Concerns) seems to not say about `<template src="foo.html">`.